presto-main/src/main/java/com/facebook/presto/server/ServerMainModule.java

```diff
@@ -688,7 +688,7 @@ public ListeningExecutorService createResourceManagerExecutor(ResourceManagerCon
                 .addProperty("coordinator", String.valueOf(serverConfig.isCoordinator()))
                 .addProperty("resource_manager", String.valueOf(serverConfig.isResourceManager()))
                 .addProperty("catalog_server", String.valueOf(serverConfig.isCatalogServer()))
-                .addProperty("sidecar", String.valueOf(serverConfig.isCoordinatorSidecar()))
+                .addProperty("sidecar", String.valueOf(false))
```

Comment:
Actually, I think we can just remove it altogether, if it's not present it's treated as false.